yajsapi/rest/payment.ts

```diff
@@ -196,12 +210,26 @@ export class Payment {
     _allocation._api = this._api;
     _allocation.id = allocation_obj.allocationId;
     _allocation.amount = parseFloat(allocation_obj.totalAmount);
+    _allocation.payment_platform = allocation_obj.paymentPlatform;
+    _allocation.payment_address = allocation_obj.address;
     _allocation.expires = new Date(
       parseInt(allocation_obj.timeout as string) * 1000
     );
     return _allocation;
   }
 
+  async *accounts(): AsyncGenerator<yap.Account> {
+    let { data: _accounts } = await this._api.getSendAccounts();
+    for await ( let account_obj of _accounts ) {
+      yield account_obj;
+    }
+  }
+
+  async decorate_demand(ids: string[]): Promise<yap.MarketDecoration> {
+    const { data: _decorate_demand } = await this._api.decorateDemand(ids);
+    return _decorate_demand; 
+  }
+
   async *invoices(): AsyncGenerator<Invoice> {
     let { data: result } = await this._api.getReceivedInvoices();
     for (let invoice_obj of result) {
```

Comment:
In `accounts()` the loop looks like this:
`for await ( let account_obj of _accounts ) {`
but in `invoices()` it looks like this:
`for (let invoice_obj of result) {`

Reply:
nice catch, thanks! it should be without `await` in `accounts`.